source/sass/blocks/example.scss

```
@@ -0,0 +1,34 @@
+.example {
+  padding: 27px 20px 25px;
+  background-color: $example-background;
+}
+
+.example__title {
+  font-family: "Oswald Regular", "Arial Regular", sans-serif;
```

Вынести в переменную, это удобнее и быстрее когда автокомплит тебе подсказывает. У Arial убрать regular 
```suggestion
  font-family: "Oswald Regular", "Arial", sans-serif;
```

---

почему-то не работает автокомплит для переменных